tests/features/project.feature

```diff
@@ -38,18 +61,51 @@ Feature: Project content creation
     And I fill in "Alternative title" with "My alternative title text" in the "Alternative titles and teaser" region
     And I fill in "Navigation title" with "My navigation title text" in the "Alternative titles and teaser" region
     And I fill in "Departments" with "Audit Board of the European Communities"
-
-    # Add link fields.
+    
     When I press "Save"
     Then I should see "My Project"
     And I should not see "EU financing"
-    And I should not see "Committee on Agriculture and Rural Development"
     And I should see "European Patent Office"
-    And I should see "Audit Board of the European Communities"
-    And I should see "Anti Fraud Information System (AFIS)"
-    And I should not see "Project teaser"
-    And I should see "Summary text"
     And I should see "Body text"
+    # Coordinators
+    And I should see "Coordinators stakeholder"
+    And I should see "CS"
+    And I should see "France"
+    And I should see "Ma société 1"
+    And I should see "1 rue de la Paix"
+    And I should see "Etage 1"
+    And I should see "11111"
+    And I should see "Cruscades"
+    And I should see "1"
+    And I should see "https://ec.europa.eu/contact1"
+    And I should see "Logo"
+    And I should see "https://ec.europa.eu/1"
+    # Participants
+    And I should see "Participants stakeholder"
+    And I should see "PS"
+    And I should see "Luxembourg"
+    And I should see "Ma société 2"
+    And I should see "2 Avenue du Blues"
+    And I should see "Etage 2"
+    And I should see "2222"
+    And I should see "Luxembourg city"
+    And I should see "https://ec.europa.eu/contact2"
+    And I should see "Logo"
```

Comment:
This step is already present on the top portion, so we have two times 
```
     And I should see "Logo"
...
     And I should see "Logo"
```
And we don't know to which logo we are referring. I'd suggest to add a dedicated Behat region representing each field, and then we can assert: 
```gherkin
     And I should see "Logo" in the "Project participants" region
```
Same for the coordinators.

Reply:
done